### PR TITLE
Update branding for Bedrock site

### PR DESCRIPTION
**Pull request by shawnborton**
### Details
This updates the branding found on the bedrockdb.com website. 

image

image

### Fixed Issues
Fixes [GH_LINK](https://github.com/Expensify/Expensify/issues/192915)

### Tests
1. Tested locally by running `jekyll serve` from `Bedrock/docs`
_________
**Internal Testing Reminder:** when changing bedrock, please compile auth against your new changes

### CODE REVIEW

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -4,551 +4,494 @@
...
...KpyoZ3cVjKbEcr2mSgoAbhh6iN4a1fdWUMjHEj1PZQow7ce4Edg4hMrXREyguQ3bsOJdrYqlqRo2TYsnqKfh2dRyocUo1Hk6pBLQ/iEMqCHZl5xeXV6DMKZTkCTXKlJDk6BkfiU6ltDkV7Bh5Im2ufKCGjrEAGRWK4iAuxQni4LMPsUjnpEz3EfOM3Xz9goVf/OtFCx+ZWzf/4Xn1Ow93bNnz1c8vXX14546fkifvI3Fhh2Ve3ZyHEg+umlk9bVbdI5amLc0r5zSazI2LVrduGThzP9mDnAmoUeGI6KW9wO1cspR6TODOFYENMtQJrPVu6VnCWzbZe+ymsDWSCrJm4GA03RZsYoaK38bagoNoqFgFGm7qEXOdRYggkx4QZKHtweA2YVKVM+SxYkDvZGwEnkbidJAUrZihSAt7NUma3RgcOPV4tKx76rTG+XO2tGzcnfua+e7ly7fI5xdMpbHILywYoye46sSvLogvajFK5H3aL6q/wRmA+533d4xahjpGXVHkVk4hFlrdmNUxGvfiy8iu0RdP93b+aGTnqP7G9YFZWvMoN/z+ttHubxzl/mN1rAqgDEfrWq189dTlkZ2rZMv16yPub+U899/fNnT/dOKF2Ki5k3X/BCAgbBgFBdvOXX1T3rTpyfuwoL9+fffuLESk13ET1hGE48+PXEdJeh1InDlWTDD3SDlFxoiSBydBsIjGR5UIXV4JLK+8RpHyNHmYWeZQaH1YiFI/2sobrc1Lo3MqghX5ZZXWDdrJOD+cWO4DxrdidZHf66qs8qxYE8jHg4G5GmSiBtc7AJeTxinXj4TMlYFMispl8VSOyE0AZgrEqMNtvJKyOjg/XLCy+XYl9IxGL61GNDhcNRhg6iF8no9GMR2Kzo3MRbAnPC8b/uyCKA3i7GtZ0K+cU1U1B3+eSAO7ezaczY5Vzc5AvZqdzx6waRT1e+0Csz8o3IatlK5HhTtD2WgNBpjQLxuyBgF2+5VUAVOeLGybtg3TjU6vED7X4nNSoawxhVIWwAgBTYTdxx5jDe0bhWWOtzY2tuJPc200+uCD0WjtSAbiaxqamxvwp2ratKqq6dM5/t57HKe/QXvBHdxyZkXIXHx4E7DVjoYDyEXszbVkenOdaM/LUizTnmtJFwtkGoNZmZ2oA0AdAF6IDLUGw49AgpJL+Cth7+A6/qnBH/A/HjigcudIkHR
```
Comment (NikkiWines) [inline docs/_layouts/default.html:40]: Any reason for the intense indentation here (and for the other lines)?

**Review by NikkiWines — CHANGES_REQUESTED** [review]
Looks great, one note is that at a smaller viewport size it looks like we still get the gradient

image

File: docs/_layouts/default.html (hunk 1 above)
Comment (shawnborton) [reply docs/_layouts/default.html:40]: No particular reason, I guess it just lines up perfectly with the `url` above?

**Comment by shawnborton** [comment]
Fixed mobile and made the nav menu friendlier, currently it gets clipped:
image

**Review by NikkiWines — APPROVED** [review]
👍  Looks great! Thanks @shawnborton

**Pull request merged**